_posts/Illumination_Correction_A_Comparison_of_Methods.md

```diff
@@ -0,0 +1,135 @@
+Conveniently, the workflow is the exact same. 
+I was able to take from the examples, load in my images, and produce illumination corrected images (see https://github.com/WayScience/NF1_SchwannCell_data/blob/main/1_preprocessing_data/PyBaSiC_Pipelines/Illumination_Correction.ipynb).
+Even though it was so easy, the hardest part was the fact that the package did not have a saving function.
+
+I was able to ask the developers on GitHub (see https://github.com/peng-lab/BaSiCPy/issues/91) what the best form of saving the images was and developers answered within a week, which was great! 
+This package is very well-maintained and since it is newer, that means that it can only improve. 
+I did find however that I struggled with figuring out the format of what my new images were converted into (i.e. 32-bit, 64-bit, etc.), which is important for the next pipeline step (segmentation) (that I will discuss in a future blog post). 
+One of the positives for this method in the context of a new biologist in this field is that the “correct_illumination” and “basic” function already have established parameters, which seems robust for a variety of use cases, that you do not need to change. 
+For more information on how PyBaSiC compares with other illumination correction methods (spoiler alert, it seems to work WAY better) or to learn the math behind how it calculates the flatfield and darkfield functions for correction, you can read the [BaSiC paper](https://www.nature.com/articles/ncomms14836).
+
+For PyBaSiC, the only thing you need to toggle is loading in the images and the best way to save your newly corrected images. Depending on your project, maybe the format of the corrected images is fine for your next steps, but in other cases you might need to convert them to 8-bit or 16-bit. For my project, I implemented a [fellow lab member’s code](https://github.com/WayScience/mitocheck_data/blob/main/1.preprocess_data/preprocess_training_data.ipynb), where he converted the corrected images to 8-bit when using this method.
+I needed to use this conversion because downloading the images as-is (without conversion) was causing multiple errors during my downstream processes. 
+It is important to know that you will likely have to retrace your steps back to previous pipelines, like IC, when issues need to be corrected or to be improved. 
+In all, I believe this method is the most efficient and easy to work with to perform illumination correction.
+
+Lastly, I checked out the Fiji/ImageJ plugin called CIDRE. 
+I don’t have a lot to say about it since when I used it with my image set of 96 images, it came up with an error. 
+After further investigation, it was [an error](https://github.com/smithk/cidre/issues/3) that was found back in 2018 that has not been solved. Though, from an outsider’s perspective, this could seem like the developers abandoned the project, but this might not be true. 
```

Comment:
```suggestion
After further investigation, it was [an error](https://github.com/smithk/cidre/issues/3) that was found back in 2018 that has not been solved.
Though, from an outsider’s perspective, this could seem like the developers abandoned the project, but this might not be true. 

```